bin/test-browser.js

```diff
@@ -170,7 +170,7 @@ async function startTest() {
       return;
     }
 
-    console.log('Unhandled error in test page:', err);
+    console.log('Unhandled error in test page:', err.stack);
```

Comment:
Maybe do both. The message _might_ have useful info sometimes?

Reply:
Seems possible.. apparently `.stack` isn't specced.  Adding `.cause` might be useful too.